components/01-visual-styling/04-utility/06-misc/misc.hbs

```diff
@@ -1,9 +1,61 @@
 <div class="css-box">
-    <p>Top Image Clip : <span class="css-class">.clip-top-img</span></p>
     <p>Right Image Clip : <span class="css-class">.clip-mask-right-bottom</span></p>
     <p>Left Image Clip : <span class="css-class">.clip-mask-left-bottom</span></p>
 </div>
 
+<section class="py-5 white-bg">
+    <div class="container">
+        <div class="row">
+            <div class="col">
+                <h3>Top Left Mask Clip</h3>
+                <p>
+                    <span class="css-class">.clip-mask-top-left</span>
+                </p>
+                <img src="{{ path mask.image }}" class="clip-mask-top-left" alt="Card Image" />
+            </div>
+            <div class="col">
+                <h3>Top Right Mask Clip</h3>
+                <p>
+                    <span class="css-class">.clip-mask-top-right</span>
+                </p>
+                <img src="{{ path mask.image }}" class="clip-mask-top-right" alt="Card Image" />
+            </div>
+        </div>
+        <div class="row">
```

Comment:
It might be nice to add some top margin, so people don't get confused about what is what.